ansible/roles_ocp_workloads/ocp4_workload_rhtr_xraylab/templates/odh-operator/subscription.yaml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: opendatahub-operator
+  namespace: openshift-operators
+spec:
+  channel: beta
+  installPlanApproval: Automatic
+  name: opendatahub-operator
+  source: community-operators
```

Comment:
You should probably add catalog snapshot support here.

Reply:
Will do

Reply:
Done!